src/OpenLoco/Localisation/StringManager.h

```diff
@@ -31,38 +31,38 @@ namespace OpenLoco
     namespace ControlCodes
     {
         // Arguments (1 byte): uint8_t
-        constexpr uint8_t move_x = 1;
-        constexpr uint8_t adjust_palette = 2; // Not used
+        constexpr uint8_t moveX = 1;
+        constexpr uint8_t adjustPalette = 2; // Not used
         // 3-4 Not used
-        constexpr uint8_t oneArgBegin = move_x;
+        constexpr uint8_t oneArgBegin = moveX;
         constexpr uint8_t oneArgEnd = 4 + 1;
 
         // Arguments: none
         constexpr uint8_t newline = 5;
-        constexpr uint8_t newline_smaller = 6;
-        constexpr uint8_t font_small = 7;
-        constexpr uint8_t font_large = 8;
-        constexpr uint8_t font_bold = 9;
-        constexpr uint8_t font_regular = 10;
+        constexpr uint8_t newlineSmaller = 6;
+        constexpr uint8_t fontSmall = 7;
+        constexpr uint8_t fontLarge = 8;
+        constexpr uint8_t fontBold = 9;
+        constexpr uint8_t fontRegular = 10;
         constexpr uint8_t outline = 11;
-        constexpr uint8_t outline_off = 12;
```

Comment:
Maybe make a Font namespace for these as well?

Reply:
I thought about it haha but in the end didn't. Happy to do that when I get home.

Reply:
done